docs/README.md

```diff
@@ -0,0 +1,8 @@
+# How to generate this document locally
+
+- Clone this repository and install the required dependencies through poetry.
```

Comment:
I think you should add the command needed to to this because the dependencies for building the docs are defined as optional.

Reply:
Also, not sure why we need pip to install those dependencies if they can be installed with poetry.

Reply:
ok. It makes sense. I updated the docs with the appropriate instructions.